MetamodelAlgemeen.md

```diff
@@ -352,6 +352,28 @@ Meervoudige overerving of multiple-inheritance:
 
 Een subtype kan meerdere objecttypen als generalisatie hebben. In het diagram [Kern](#kern) is dit aangegeven door een `«Objecttype»` als subtype naar `0..*` `«Generalisaties»` te laten verwijzen. Dat impliceert dat een subtype `0..*` supertypen kan hebben.
 
+#### Mixin
+
+<aside class="definition">
+  <dfn>Mixin</dfn>Metagegeven om bij een generalisatie aan te geven dat bij een implementatie die geen multiple inheritance ondersteund de eigenschappen van de superklasse worden overgenomen door de subklasse. De superklasse zelf komt niet in de implementatie voor. 
+</aside>
+
+*Toelichting:*
+Het metagegeven `Static`
```

Comment:
moet dit geen Mixin zijn of heb ik iets gemist?

Reply:
Nu zie ik het . Mixin en de tekst over static staat hier nog bij de modelelementen. Daar moet het weg. Het is immers bij de metagegevens opgenomen.